gulp/gulp-tests.js

```diff
@@ -235,14 +235,22 @@ gulp.task('test', gulp.series(
   'test:sanity',
   'test:content',
   'test:common',
-  'test:api:unit',
+  'test:prepare:mongo',
+  'test:api:unit:run',
```

Comment:
This one can still reference the new series right? (And the same for test:api-v3)